net-misc/ntp/ntp-4.2.8_p15-r1.ebuild

```diff
@@ -109,7 +109,7 @@ src_install() {
 	sed -i "s:/usr/bin:/usr/sbin:" "${ED}"/etc/init.d/ntpd || die
 
 	keepdir /var/lib/ntp
```

Comment:
Please drop the keepdir and fowners and use a tmpfiles.d snippet to create the directory instead.